aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -709,6 +710,27 @@ impl AptosVM {
 
         let func = session.load_script(script.code(), script.ty_args())?;
 
+        // Check that unstable bytecode cannot be executed on mainnet
+        if self
+            .features()
+            .is_enabled(FeatureFlag::REJECT_UNSTABLE_BYTECODE_FOR_SCRIPT)
```

Comment:
Do we need to check both `REJECT_UNSTABLE_BYTECODE_FOR_SCRIPT` and `REJECT_UNSTABLE_BYTECODE` given that `REJECT_UNSTABLE_BYTECODE` is already enabled?

Reply:
I think a new flag only is enough